src/ducks/connections/index.js

```diff
@@ -27,11 +27,12 @@ export const END_CONNECTION_CREATION = 'END_CONNECTION_CREATION'
 const getTriggerKonnectorSlug = trigger =>
   (trigger && trigger.message && trigger.message.konnector) || null
 
-const isKonnectorTrigger = doc =>
+export const isKonnectorTrigger = doc =>
   doc._type === TRIGGERS_DOCTYPE && !!doc.message && !!doc.message.konnector
 
 const isKonnectorJob = doc =>
-  doc._type === JOBS_DOCTYPE && doc.worker === 'konnector'
+  doc._type === JOBS_DOCTYPE &&
+  (doc.worker === 'konnector' || doc.worker === 'client')
```

Comment:
I've done a quick search in harvest. Do we need to update this method for instance https://github.com/cozy/cozy-libs/blob/9ba0a2453ad7640c2046a27b649b29c82a7c5303/packages/cozy-harvest-lib/src/services/utils.js#L79?

Reply:
Yes https://github.com/cozy/cozy-libs/commit/f8f2a084181e44dddcc76e88ad3fe23401e57430